src/views/TabView/TabView.js

```diff
@@ -93,11 +93,30 @@ class TabView extends React.PureComponent {
       tabBarOptions,
       tabBarComponent: TabBarComponent,
       animationEnabled,
+      navigation,
+      router,
+      screenProps,
     } = this.props;
     if (typeof TabBarComponent === 'undefined') {
       return null;
     }
 
+    const { state } = navigation;
+    const options = router.getScreenOptions(
+      this.props.childNavigationProps[state.routes[state.index].key],
+      screenProps || {}
+    );
+
+    const tabBarVisible =
+      options.tabBarVisible === null || options.tabBarVisible === undefined
```

Comment:
`options.tabBarVisible === false ? false : true`

Reply:
Ok, is there a reason why to not use `const tabBarVisible = options.tabBarVisible !== false;`

Reply:
that's fine too, the current form is just a bit verbose

Reply:
Updated!